source/agora/test/NetworkManager.d

```diff
@@ -178,6 +179,12 @@ unittest
     node_bad.clearFilter();
     node_validators.each!(node => node.clearFilter());
 
+    // The testing time decreases compared to the last code because setting
+    // seeds to the `PreImageCycle`s make it unnecessary to generate kind of
+    // large number seeds. So we have to add some preparation time to this
+    // network test like this.
+    Thread.sleep(5.seconds);
+
```

Comment:
Shouldn't this be removed in the second commit, which I guess will be squashed into first?

Reply:
Currently, it should remain because the test gets to need some waiting time with the pre-image generating time decreasing. I think that the `catchupTask` starts kind of late after applying the code of this PR.